test/functional/test_framework/test_framework.py

```diff
@@ -541,15 +539,52 @@ def join_network(self):
         connect_nodes(self.nodes[1], 2)
         self.sync_all()
 
-    def sync_blocks(self, nodes=None, **kwargs):
-        sync_blocks(nodes or self.nodes, **kwargs)
-
-    def sync_mempools(self, nodes=None, **kwargs):
-        sync_mempools(nodes or self.nodes, **kwargs)
-
-    def sync_all(self, nodes=None, **kwargs):
-        self.sync_blocks(nodes, **kwargs)
-        self.sync_mempools(nodes, **kwargs)
```

Comment:
why is this changed?

Reply:
I removed `**kwargs` which are not used.

Reply:
I think they might come in handy in the future. Is there a reason they will be unused forever?

Reply:
I feel it's better to add it when it will be used in instead of putting it there up front just because some time in the future it might be used. What do you think?